frontend/src/scenes/experiments/SecondaryMetrics.tsx

```diff
@@ -195,7 +195,7 @@ export function SecondaryMetrics({
                                         mathAvailability={MathAvailability.None}
                                         buttonCopy="Add funnel step"
                                         seriesIndicatorType="numeric"
-                                        sortable
+                                        sortable={true}
```

Comment:
No preference really. I just came across it as I in the same area. Made the switch because most other props were explicitly setting the value. Right approach is to remove it from all of them 😂 We should really just set the linter to be strict on that